Minor tweaks to B2B metrics (#4520)
PostHog/posthog.com: contents/blog/b2b-saas-product-metrics.md

---

> date: 2022-10-25

Author: ```suggestion
date: 2022-10-26
```

title: The most useful B2B SaaS product metrics
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
featuredImage: ../images/blog/b2b-product-metrics/b2b-product-metrics.jpeg
featuredImageType: full
categories: ["Guides"]
author: ["andy-vandervell"]
---

So, you're building a B2B SaaS product.

Everyone knows you've got to measure stuff to succeed. This is not news. 

What, why, and how? These are the questions that matter and the answers lie in the product metrics you track.

In this guide, we'll cover:

1. [What makes a good metric](#what-makes-a-good-b2b-product-metric) for a B2B product
2. [The common metrics used by B2B SaaS companies](#common-b2b-saas-product-metrics)
3. [How to choose your product metrics](#how-to-choose-your-product-metrics) 

> > **Note:** This guide is about _product_ metrics, not revenue or wider business metrics like customer lifetime value (CLV) or average revenue per user (ARPU). Naturally, they're important too, but this guide is all about the metrics of most interest to product teams.

Reviewer: ```suggestion
> **Note:** This guide is about _product_ metrics, not revenue or wider business metrics like customer lifetime value (LTV) or average revenue per user (ARPU). Naturally, they're important too, but this guide is all about the metrics of most interest to product teams.
```

Reviewer: I think LTV is a more standard acronym for lifetime value.

## What are product metrics?

Product metrics capture how people use your product **and** how that impacts your business. 

> For example, if the number of new users who go on to successfully use your product (aka new user activation) is declining, you can expect this to lead to a decline in the number of paying users.

Reviewer: ```suggestion
For example, if the number of new users who successfully use your product (aka new user activation) declines, you can expect this to lead to a decline in the number of paying users.
```

What makes this a good product metric for us? Let's break it down:

- **Is easy to understand?** Yes. While it combines several different events into one action, they're all easy for anyone who knows the product to understand.

- **Is it comparable?** Absolutely. We can use this metric to compare power users to average users and compare different cohorts based on multiple criteria (e.g. paying vs non-paying customers, job title etc.). It's also sensitive to changes in the product, so we can see whether our users make more or fewer discoveries when we make a change.

- **Is it specific?** Yes. Again, while it's a composite metric, each individual event captures users getting value from our product.

- **Is it actionable?** Yes. If this metric drops, it immediately gives us several avenues of investigation to understand the cause.

Not all useful product metrics follow this trend – active users is a notable exception – but it's a good acid test to apply to any metric you're considering.

> ### Recommended B2B product metrics

Reviewer: Don't we kind of already say this section in the `Common metrics` above? Should we just combine the two?
